.github/scripts/run_torchbench.py

```diff
@@ -31,6 +33,25 @@
 direction: decrease
 timeout: 720
 tests:"""
+S3_BUCKET = "ossci-metrics"
+S3_PREFIX = "torchbench-pr-test"
+S3_URL_BASE = f"https://{S3_BUCKET}.s3.amazonaws.com/"
+
+class S3Client:
+    def __init__(self, bucket: str = S3_BUCKET, prefix: str = S3_PREFIX):
+        self.s3 = boto3.client('s3')
+        self.resource = boto3.resource('s3')
+        self.bucket = bucket
+        self.prefix = prefix
+
+    def upload_file(self, file_path: Path, filekey_prefix: str) -> None:
+        assert file_path.is_file(), f"Specified file path {file_path} does not exist or not file."
+        file_name = file_path.name
+        s3_key = f"{self.prefix}/{filekey_prefix}/{file_name}"
```

Comment:
Is `filekey_prefix` unique or would it could repeated over multiple runs of the same PR or the same git commit?

Reply:
It is the same for the same PR, the second run will overwrite the result of the first run.

Reply:
Let me know if you need to keep the run history. If you do, we probably can do it in a follow-up PR.